docs/tarball-readme.md

```diff
@@ -0,0 +1,11 @@
+# OpenTelemetry Collector for Google Cloud Monitoring Tarball
+
+Thank you for using this OpenTelemetry Collector and Google Cloud Monitoring. This packaged tarball contains tools to immediately setup the Collector with compatible metrics for your Google Cloud Monitoring.
+
+## Getting Started
+1. Run `bash run.sh` to run the collector
+2. Search for metrics in the Google Cloud Monitoring project, there should already be values populated!
+
+## Customed Metrics
+
+Modify the configuration file `config-example` to experiement with different metrics
```

Comment:
How about as part of the tarball generation we rename this to just `config.yaml`?